src/components/Schema/ArraySchema.tsx

```diff
@@ -23,6 +24,7 @@ export class ArraySchema extends React.PureComponent<SchemaProps> {
     return (
       <div>
         <ArrayOpenningLabel> Array ({minMaxItems})</ArrayOpenningLabel>
+        <TypeTitle>{itemsSchema.schema.title}</TypeTitle>
```

Comment:
what if there is no title?

Reply:
Good call, set the block to only render if the schema title exists.